homeassistant/components/switch/broadlink.py

```diff
@@ -96,20 +100,20 @@ def _learn_command(call):
 
         _LOGGER.info("Press the key you want Home Assistant to learn")
         start_time = utcnow()
+        title = friendly_name + " Learned "
         while (utcnow() - start_time) < timedelta(seconds=20):
             packet = yield from hass.async_add_job(
                 broadlink_device.check_data)
             if packet:
-                log_msg = "Recieved packet is: {}".\
-                          format(b64encode(packet).decode('utf8'))
+                data = format(b64encode(packet).decode('utf8'))
+                log_msg = "Recieved packet is: " + data
                 _LOGGER.info(log_msg)
-                hass.components.persistent_notification.async_create(
-                    log_msg, title='Broadlink switch')
+                _save_packet(call, data, title)
```

Comment:
We do not want to write it to disk. Revert this and keep the persistent_notification

Reply:
i have no idea where to save the packet info
and i saw the `device_tracker` save the info in `known_devices.yaml` 
so i think it's right to create a new yaml file
otherwise,which is better to save data like this.
please give me some suggestion.

Reply:
Why do we need to write it to file?
Is that better than the system we have?

Reply:
In the original `broadlink.py`,
the packet data on the frontend 
shown by `persistent_notification` cannot be copyed.
we only copy them from logger .
Becasue they are too similar ,
manage without ID is a problem .
so, I advice to save them to a file and
use `broadlink_send_packet_by_id` read file,
it's a simple solution to create a script/automation.

Reply:
I have no idea about how to FIX `persistent_notification` 
maybe add a COPY button or something else
that's out of my ability